pytorch/inference/docker/1.10/py3/Dockerfile.graviton

```diff
@@ -96,11 +95,22 @@ RUN conda install -c \
     "cryptography>=3.3.2"
 
 # install torch and torchvision from the PyTorch website
-RUN pip install torch torchvision
+RUN pip uninstall -y torch \
+ && pip install --no-cache-dir -U $PT_INFERENCE_URL
+RUN pip install torchvision
+
+## compile and install ArmCL
+RUN git clone https://github.com/Arm-software/ComputeLibrary.git \
+ && cd ComputeLibrary \
+ && git checkout $ARMCL_BRANCH \
+ && scons Werror=0 -j8 debug=0 neon=1 opencl=0 os=linux arch=arm64-v8a build=native \
```

Comment:
lets also remove `scons` after the ArmCL binary is generated as it adds no value to the image after that.

Reply:
> > > > Are we planning to tag these as e3?
> > > 
> > > 
> > > you mean tag this PR or the DLC as e3?
> > 
> > 
> > I mean tag the DLC as e3
> 
> I think it can be tagged as e3, @tejaschumbalkar what do you think?

lets create a separate PR for DIY/SM split.